pkg/sql/catalog/descs/collection.go

```diff
@@ -282,6 +284,29 @@ func (tc *Collection) GetAllDatabaseDescriptors(
 	return tc.kv.getAllDatabaseDescriptors(ctx, txn)
 }
 
+func (tc *Collection) GetAllDescriptorsInDatabase(
```

Comment:
Add a comment for this method. Can this be GetAllTableDescriptors? And the return type []catalog.TableDescriptor instead?